build-logic/convention-plugins/build.gradle.kts

```diff
@@ -2,15 +2,11 @@ plugins {
     `kotlin-dsl`
 }
 
-gradlePlugin {
```

Comment:
Rolled the code-quality config into the common convention plugin, since we should apply it to all modules anyway. Could keep it separate if there's a huge desire to though.

Reply:
It seems we lost `spotless` from `app-*` modules, so maybe we could have another plugin named `.app` where we apply it there?

Reply:
Fixed